### DISCUSSION TITLE
👀 Angular 17 Support?

### DISCUSSION THREAD

**Discussion by WuglyakBolgoink** [salesforce/akita#1076] [Q&A]
Hi, I'm in the process of upgrading some apps to Angular 17. Does Akita work with v17? Are there any plans to support it? Thank you!

**Comment by arizmendiomar** [comment]
Any news about this topic?

**Comment by mirumirumi** [comment]
I have used Akita 3 times on a clean Angular 17 project with no problems at all. Everything is working fine!

**Reply by WuglyakBolgoink** [reply]
@mirumirumi  the question is: WHY?)

> THE LIBRARY IS NOT MAINTAINED ANYMORE - DON'T USE IT

**Reply by jahller** [reply]
it works with Angular 17, but i would not recommend to still use it.
rather use ngrx.
